templates/default/less/variables.less

```diff
@@ -343,3 +340,17 @@
 //##
 @il-counter-novelty-bg: @brand-warning;
 @il-counter-novelty-margin-top: -5px;
+
+//== Icons
+//
+//## Icons are used to represent objects (and quickly identify) objects..
+//**Size for icons: Small is used in the tree. e.g.
+@icon-size-small: 20px;
+//**Size for icons: Medium is used in the content-listing. e.g.
+@icon-size-medium: 35px;
+//**Size for icons: Large is used in the header. e.g.
+@icon-size-large: 45px;
+//**std. icons
```

Comment:
Add those comments per variable, do not treat them as bulk. E.g. Std icon for groups.